src/content/en/fundamentals/performance/speed-tools/images/tool-webpagetest.svg

```diff
@@ -0,0 +1 @@
1-29.67l11.76 11.76 2.56-2.56-11.58-11.58a43 43 0 0 1 25-9.32v16.55h3.62V73.9a43 43 0 0 1 25.39 9.62L161.44 94.8l2.56 2.55 11.43-11.43a43.09 43.09 0 0 1 13.26 29.34h-15.82v3.62h15.82a43.05 43.05 0 0 1-6.64 21.28l3.06 1.93a46.85 46.85 0 0 0-39.6-71.86z"/><path class="cls-2" d="M153.09 117.64a7.58 7.58 0 1 0-1.83 4.92L176.84 135l1.58-3.25-25.53-12.38a7.58 7.58 0 0 0 .2-1.73z"/><path class="cls-3" d="M52.34 198.55l-4.83-40h4.17l3 27.5 3.68-27.5h4.85l3.54 28 3.11-28H74l-4.89 40h-4.36l-4-29.93-4 29.93zm24.84 0v-40h13.65v6.77h-9.95v8.87h9.24v6.74h-9.24v10.86h10.29v6.77zm18.43-40.02h8.47q3.47 0 4.88 1a7.34 7.34 0 0 1 2.38 3.47 16.85 16.85 0 0 1 .14 10.85 7.85 7.85 0 0 1-2.15 3.45 6.94 6.94 0 0 1 2.94 3.55 15.68 15.68 0 0 1 1.06 6.11 18 18 0 0 1-1.35 7.38q-1.35 3-3.2 3.73a21.57 21.57 0 0 1-6 .49h-7.17zm4.26 6.66v9.25h2.81a20.14 20.14 0 0 0 3.13-.14 2.63 2.63 0 0 0 1.71-1.43 6.76 6.76 0 0 0 .62-3.15 7 7 0 0 0-.5-2.88 2.43 2.43 0 0 0-1.21-1.38 16.81 16.81 0 0 0-4.09-.27zm0 15.91v10.67h4a8.19 8.19 0 0 0 3.35-.42 3.2 3.2 0 0 0 1.24-1.66 8.45 8.45 0 0 0 .5-3.15 8.33 8.33 0 0 0-.54-3.23 3.35 3.35 0 0 0-1.43-1.74 9 9 0 0 0-3.64-.48zm16.59 17.45v-40h6a10.55 10.55 0 0 1 4.36.57q1.64.93 2.7 4a24 24 0 0 1 1.06 7.81 24.68 24.68 0 0 1-.92 7.28 8.88 8.88 0 0 1-2.29 4.12q-1.38 1.19-4.74 1.19h-2.43v15.1zm3.7-33.25v11.38h2.05a5.3 5.3 0 0 0 2.81-.52 3.71 3.71 0 0 0 1.22-1.87 11.81 11.81 0 0 0 0-6.69 3.51 3.51 0 0 0-1.21-1.83 6.46 6.46 0 0 0-3.05-.47zm28.16 33.25h-4l-1.61-9.06h-7.36l-1.52 9.06h-3.95l7.16-40h3.93zm-6.83-15.83l-2.54-14.85-2.49 14.85zm14.93 1.14v-6.77h8V193a12.67 12.67 0 0 1-3.42 4.4 7 7 0 0 1-4.45 1.83 6.22 6.22 0 0 1-4.8-2.39 17.41 17.41 0 0 1-3.3-7.15 45.62 45.62 0 0 1-1.22-11.21 45.06 45.06 0 0 1 1.23-11.3 17.92 17.92 0 0 1 3.2-7 6 6 0 0 1 4.71-2.32 6.06 6.06 0 0 1 5.27 3q2 3 2.55 8.78l-3.67 1.5a11.21 11.21 0 0 0-1.52-4.72 3.16 3.16 0 0 0-2.63-1.64q-2.37 0-3.84 3.29t-1.47 10q0 7.23 1.59 11 1.36 3.25 3.69 3.25a3.71 3.71 0 0 0 2.26-.91 8.53 8.53 0 0 0 2.09-2.49v-5.06zm10.25 14.69v-40h13.65v6.77h-9.95v8.87h9.24v6.74h-9.24v10.86h10.28v6.77zm21.95 0v-35.3h-6.06v-4.72h14.58v4.72h-6.09v35.3zm9.83 0v-40h13.3v4.72h-10.86v12.23h10.18v4.72h-10.18v13.62h11.29v4.72zm15.07-12.89l2.3-.44a19.32 19.32 0 0 0 .83 5.15 7.21 7.21 0 0 0 1.86 3 3.9 3.9 0 0 0 2.72 1.12 3.79 3.79 0 0 0 3.37-1.89 9.06 9.06 0 0 0 1.2-4.92 9.89 9.89 0 0 0-.44-3.09 5.21 5.21 0 0 0-1.32-2.17 19.34 19.34 0 0 0-3.8-2.39 11.66 11.66 0 0 1-3.7-2.77 10.09 10.09 0 0 1-1.71-3.54 17.63 17.63 0 0 1-.61-4.84 15.9 15.9 0 0 1 1.78-7.98 5.31 5.31 0 0 1 4.74-3.08 5.07 5.07 0 0 1 3.59 1.45 9.46 9.46 0 0 1 2.39 4.15 22.56 22.56 0 0 1 .91 6.36l-2.34.38a12.81 12.81 0 0 0-1.31-5.79 3.71 3.71 0 0 0-6.31-.23 7.64 7.64 0 0 0-1.09 4.27 7.4 7.4 0 0 0 .8 3.77 8.46 8.46 0 0 0 3.58 2.71 18.1 18.1 0 0 1 3.82 2.33 9.52 9.52 0 0 1 2.54 4 18.36 18.36 0 0 1 .86 6 20.48 20.48 0 0 1-.9 6.26 10 10 0 0 1-2.46 4.29 5.24 5.24 0 0 1-3.74 1.47q-3.41 0-5.44-3.66t-2.12-9.92zM235 198.55v-35.3h-6.06v-4.72h14.58v4.72h-6.09v35.3z"/></g></g></svg>
```

Comment:
please check if this icon exists anywhere else on the site, and use that if possible.